src/plugins/saved_objects_finder/public/finder/index.tsx

```diff
@@ -8,7 +8,12 @@
 
 import { EuiDelayRender, EuiLoadingContent } from '@elastic/eui';
 import React from 'react';
+import { IUiSettingsClient } from '@kbn/core-ui-settings-browser';
+import { HttpStart } from '@kbn/core-http-browser';
+import { SavedObjectsManagementPluginStart } from '@kbn/saved-objects-management-plugin/public';
+import { SavedObjectsTaggingApi } from '@kbn/saved-objects-tagging-oss-plugin/public';
 import type { SavedObjectFinderProps } from './saved_object_finder';
+import SavedObjectFinderUi from './saved_object_finder';
```

Comment:
I'm making this difficult, sorry 😅. It looks like there's still a 4kb bundle increase and I _think_ it's because of this import which is no longer used since we switched to the lazy one for `getSavedObjectFinder`. Could we remove this import and its export below and see if that fixes the bundle size increase? If it doesn't fix it, I'll give up and approve anyway -- just want to see if that does it first.

Reply:
I think it should be fine now